modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -487,36 +485,36 @@ both
 
 == Next Steps
 
-Now you're up and running, try one of the following:
+Now that your setup is up and running, try one of the following:
 
-* Our xref:hello-world:sample-application.adoc[Travel Sample Application] demonstrates all the basics you need to know;
-* Explore xref:howtos:kv-operations.adoc[Key Value Operations] (CRUD) against a document database;
-* Or xref:howtos:sqlpp-queries-with-sdk.adoc[Query] with our SQL-based {sqlpp} query language;
-* Try longer-running queries with our xref:howtos:analytics-using-sdk.adoc[Analytics Service];
-* A xref:howtos:full-text-searching-with-sdk.adoc[Full Text Search];
-* Or read up on xref:concept-docs:data-services.adoc[which service fits your use case].
+* Our xref:hello-world:sample-application.adoc[Travel Sample Application] demonstrates all the basics you need to know.
+* Explore xref:howtos:kv-operations.adoc[Key Value Operations] (CRUD) against a document database.
+* xref:howtos:sqlpp-queries-with-sdk.adoc[Query] with our SQL-based {sqlpp} query language.
+* Try longer-running queries with our xref:howtos:analytics-using-sdk.adoc[Analytics Service].
+* A xref:howtos:full-text-searching-with-sdk.adoc[Full Text Search].
+* Read about xref:concept-docs:data-services.adoc[which service fits your use case].
```

Comment:
These three bullets read as three clauses in a single sentence.
I am all for that being changed to something more consistent with our style guide, but to do so you'll need to also change the middle clause on FTS to a sentence with a verb.